README.md

```diff
@@ -279,7 +279,7 @@ Clone the Lisk Core repository using Git and initialize the modules.
 git clone https://github.com/LiskHQ/lisk.git
 cd lisk
 git checkout master
-npm install
+npm ci
```

Comment:
Due to the explanation from the comment https://github.com/LiskHQ/lisk/pull/2662#issuecomment-448969174 , we should have `npm install` here and clarify that `npm ci` is meant to be used when we are building the app not when installing. We must mention the finding in a comment to prevent users to fall in the same issue:
- `npm ci` doesn't apply the restrictions `engines` set in the `package.json`.

Reply:
It seems we have to choose between having the right packages (from `package-json.lock`) or enforcing the `engines`; it is unfortunate we cannot have both.
I would argue it is easier to document which version of npm to use (and to follow that instruction) that to ensure the right packages are installed.

Reply:
`npm ci` in the manual might look a bit odd, but it serves the purpose of having the exact same code including the one in dependencies across all of the same Lisk Core versions running from Sources, Binaries and Docker. I found this [article](https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable) convincing.


There is a high probability that the `npm ci` would fail for other versions of Node.js due to the `engine` settings in package.json of other dependencies anyway (even if Lisk Elements or Lisk Commander).